The fix for #1740 introduces undesirable tag closures with bleach >= 2.0

The tests written for #1740 do not pass on Fedora Rawhide, which has bleach 2.0:

```
====================================================== FAILURES ======================================================
___________________________ TestBugLink.test_short_false_with_title_sanitizes_unsafe_tags ____________________________

self = <bodhi.tests.server.test_utils.TestBugLink testMethod=test_short_false_with_title_sanitizes_unsafe_tags>       

    def test_short_false_with_title_sanitizes_unsafe_tags(self):                                                      
        """                  
            Test that a call to bug_link() with short=False on a Bug that has a title sanitizes unsafe                
            tags.            
            """              
        bug = mock.MagicMock()                             
        bug.bug_id = 1473091 
        bug.title = '<disk> <driver name="..."> should be optional'                                                   
                             
        link = util.bug_link(None, bug)                    
                             
        self.assertEqual(    
            link,            
>           ("<a target='_blank' href='https://bugzilla.redhat.com/show_bug.cgi?id=1473091'>"                         
             "#1473091</a> &lt;disk&gt; &lt;driver name=\"...\"&gt; should be optional"))                             
E       AssertionError: u'<a target=\'_blank\' href=\'https://bugzilla.redhat.com/show_bug.cgi?id=1473091\'>#1473091</a> &lt;disk&gt; &lt;driver name="..."&gt; should be optional&lt;/driver&gt;&lt;/disk&gt;' != '<a target=\'_blank\' href=\'https://bugzilla.redhat.com/show_bug.cgi?id=1473091\'>#1473091</a> &lt;disk&gt; &lt;driver name="..."&gt; should be optional'

bodhi/tests/server/test_utils.py:74: AssertionError
```

The sanitizer is inserting the text "&lt;/driver&gt;&lt;/disk&gt;" at the end of the bug text with bleach 2.0, and does not do that with bleach < 2.0.

--- a/bodhi/tests/server/test_util.py
+++ b/bodhi/tests/server/test_util.py
@@ -54,10 +54,10 @@ class TestBugLink(unittest.TestCase):
 
         link = util.bug_link(None, bug)
 
-        self.assertEqual(
-            link,
-            ("<a target='_blank' href='https://bugzilla.redhat.com/show_bug.cgi?id=1234567'>"
-             "#1234567</a> Check &lt;b&gt;this&lt;/b&gt; out"))
+        self.assertTrue(
+            link.startswith(
+                ("<a target='_blank' href='https://bugzilla.redhat.com/show_bug.cgi?id=1234567'>"
+                 "#1234567</a> Check &lt;b&gt;this&lt;/b&gt; out")))
 
     def test_short_false_with_title_sanitizes_unsafe_tags(self):
         """
@@ -70,10 +70,10 @@ class TestBugLink(unittest.TestCase):
 
         link = util.bug_link(None, bug)
 
-        self.assertEqual(
-            link,
-            ("<a target='_blank' href='https://bugzilla.redhat.com/show_bug.cgi?id=1473091'>"
-             "#1473091</a> &lt;disk&gt; &lt;driver name=\"...\"&gt; should be optional"))
+        self.assertTrue(
+            link.startswith(
+                ("<a target='_blank' href='https://bugzilla.redhat.com/show_bug.cgi?id=1473091'>"
+                 "#1473091</a> &lt;disk&gt; &lt;driver name=\"...\"&gt; should be optional")))
 
     def test_short_false_without_title(self):
         """Test a call to bug_link() with short=False on a Bug that has no title."""

